_posts/2023-03-02-q1-survey-results.md

```diff
@@ -0,0 +1,46 @@
+* Users identified with four roles in OpenSearch: 
+  - _Infrastructure Set Up_ - responsible for deploying OpenSearch, for allocation of resources, responsible for allocation of initial resources, and any action that impacts the billing and usage or resources
+  - _OpenSearch Admin_ - responsible for asset and user management on OpenSearch
+  - _Dashboard Creator or Data Producer_ - responsible for preparing data or dashboards for their own use or end user consumption
+  - _Dashboard or Data Consumer_ - responsible for consuming data or dashboards
+  
+* While users do not use the above labels to self-identify, the motivations described in the survey for each of these roles resonated with them.
+* 48.8%, or almost half of the community members who took the survey, self-identified themselves to be in an *administrative* role and almost half of the sample associated with all four roles.
+* 34.1% self-identified themselves to be in an *infrastructure setup* role. 
+
+Based on these insights, we concluded that close to half of our users have complex needs that span across all four roles, from *infrastructure setup* to *administrative* to *producer* to *consumer*.  Our offerings, therefore, should focus on providing consistent end-to-end workflows and transitions that enable users perform tasks seamlessly. This finding is also a good indication of the sample who accessed the survey on the OpenSearch website. In future quarters we intend to expand recruitment for the quarterly survey to the dashboard interface as well. This will help the OpenSearch team obtain feedback from dashboard users who are typically both *producers* and *consumers*.
+ 
+In Q2 we want to get a deeper understanding of ways to improve user experience on OpenSearch. We want to understand what blockers our users face when setting up OpenSearch for themselves or others. The survey will allow users to provide feedback based on their self-selected roles. We also want to understand what blockers users have when dashboards are set up, shared, and consumed. 
+
+Check out the [OpenSearch website](https://opensearch.org/) or [forum](https://forum.opensearch.org/) in March and take part in our quarterly Q2 community survey. If you are interested in participating in a deep-dive study, you’ll have the opportunity at the end of the survey to provide information on how we can reach you. As always, we welcome all community members to attend our bi-weekly [community meet-ups](https://forum.opensearch.org/t/opensearch-community-meeting-2023-0328/12531) where information on ongoing research studies are regularly announced. 
+ 
+We thank you for reading, and thanks for supporting OpenSearch!
+
+The OpenSearch UX Team
```

Comment:
```suggestion

```

Reply:
We are speaking in one voice as the OpenSearch Project—not as a UX team that is separate from the community.